api/net/Areas/Editor/Controllers/StorageController.cs

```diff
@@ -226,43 +226,37 @@ public async Task<IActionResult> Upload([FromRoute] int? locationId, [FromQuery]
     /// <returns></returns>
     [HttpGet("stream")]
     [HttpGet("{locationId:int}/stream")]
-    [ProducesResponseType(typeof(OkObjectResult), (int)HttpStatusCode.OK)]
-    [ProducesResponseType(typeof(OkObjectResult), (int)HttpStatusCode.PartialContent)]
+    [ProducesResponseType(typeof(FileStreamResult), (int)HttpStatusCode.OK)]
+    [ProducesResponseType(typeof(FileStreamResult), (int)HttpStatusCode.PartialContent)]
     [ProducesResponseType((int)HttpStatusCode.BadRequest)]
     [SwaggerOperation(Tags = new[] { "Storage" })]
-    public async Task<IActionResult> StreamAsync([FromRoute] int? locationId, [FromQuery] string path)
+    public IActionResult StreamAsync([FromRoute] int? locationId, [FromQuery] string path)
```

Comment:
This is no longer an async function, rename to `Stream`

Reply:
Updated, thanks